sensor_msgs/Image image
float32 x
float32 y